pulsar-client-kafka-compat/pulsar-client-kafka/src/main/java/org/apache/kafka/clients/consumer/PulsarKafkaConsumer.java

```diff
@@ -112,39 +113,39 @@ public PulsarKafkaConsumer(Map<String, Object> configs) {
         this(configs, null, null);
     }
 
-    public PulsarKafkaConsumer(Map<String, Object> configs, Deserializer<K> keyDeserializer,
-            Deserializer<V> valueDeserializer) {
-        this(new ConsumerConfig(ConsumerConfig.addDeserializerToConfig(configs, keyDeserializer, valueDeserializer)),
-                keyDeserializer, valueDeserializer);
+    public PulsarKafkaConsumer(Map<String, Object> configs, Schema<K> keyDeserializer,
```

Comment:
@tuteng you need to keep the old constructors. they can't be removed.

Reply:
@tuteng please don't resolve the comment if you don't address my comment. As I comment here, we need to keep the original constructors for backward compatibility.